plugins/auto-sizes/phpcs.xml.dist

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0"?>
+<ruleset name="WPP-AutoSizes">
+	<description>WordPress Coding Standards for Auto-sizes for Lazy-loaded Images Plugin</description>
+
+	<rule ref="../../phpcs.ruleset.xml"/>
+
+	<config name="text_domain" value="auto-sizes"/>
```

Comment:
Related to my previous feedback, we could add `default` as an allowed text domain for all plugins, but definitely not required as long as we don't use any core strings in them.

Reply:
I think we should keep this omitted unless we require core strings.